TiborKovari/harry-potter-react
# add home and characters pages with basic routing

### What does this implement/fix?
- Added the `Home` page (`Home.jsx`) with basic introductory content.
- Added the `Characters` page (`Characters.jsx`) to list Harry Potter characters.
- Implemented React Router for basic navigation between the two pages.
- Added a `Header` component with navigation links.

### How to test this?
1. Run `npm run dev` to start the development server.
2. Open [http://localhost:5173](http://localhost:5173) in your browser.
3. Navigate to the Home page and check the introductory content.
4. Click on the "Characters" link in the header to see the list of characters.

### Any known issues?
- Character list currently fetches all data but needs styling for better presentation.
- No additional error handling for API calls yet.
